### PR TITLE
initial reject screen changes

### PR DESCRIPTION
**Pull request by johnbradley**
This is still a work in progress and I don't intend you to merge this yet.
At this point I am just looking for feedback on the UI.

### Acknowledge Page
I added a REJECT button and changed the ACCEPT button's color:
image

Once you click “REJECT” we we show the **Reject Reason Page**.

### Reject Reason Page
image
“CANCEL” will take you back.
“REJECT” saves the reason and marks the handover with reject state then we show the **Rejection Complete Page**:

### Rejection Complete Page
image

### Model changes
I added a reject_reason TextField to Handover.

### CODE REVIEW

**Comment by hlapp** [comment]
The UI mockups look good to me.

**Comment by DukeOmicsAnalysis** [comment]
I think it looks good as well --David

**Comment by johnbradley** [comment]
I added a change to send email to the user who originally shared the project when it is accepted or rejected. If this is too much communication it will be pretty simple to remove.

**Comment by johnbradley** [comment]
@dleehr When you get some time could you review these changes?
Thanks.

File: accept/urls.py [hunk 1]
```diff
@@ -3,5 +3,6 @@
 
 urlpatterns = [
     url(r'^$', views.accept, name='accept-index'),
-    url(r'^process/$', views.accept_process, name='accept-process'),
+    url(r'^process/$', views.handover_process, name='handover-accept'),
```
Comment (dleehr) [inline accept/urls.py]: I think these names are getting confusing between accept/process/handover, and the naming isn't consistent within the app

`views.handover_process` is named `handover-accept`. should it be `handover-process`. Also, `views.accept` is named `accept-index`, but all the other views have names that start with `handover`.

Perhaps the app should be called `responsibility` or `ownership` or some other noun. Then actions like accept or reject make more sense?

File: accept/views.py [hunk 2]
```diff
@@ -1,65 +1,133 @@
...
+    Prompts for a reason for rejecting the project.
+    """
+    return render(request, 'accept/reject_reason.html', build_handover_context(handover))
+
+
+def reject_project(request, handover):
+    """
+    Marks handover rejected.
+    """
+    reason = request.POST.get('reject_reason')
```
Comment (dleehr) [inline accept/views.py]: This appears to allow a blank reason, can we require a reason?

**Pull request merged**